src/main/conversions/v2.35.0/c2_35_0_2020110601.clj

```diff
@@ -0,0 +1,28 @@
+(ns facepalm.c2-35-0-2020110601
+  (:use [kameleon.sql-reader :only [load-sql-file]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.35.0:20201106.01")
```

Comment:
I'll need to look this over more thoroughly this afternoon, but I did want to quickly mention this. There should be a corresponding line in `src/main/data/999_version.sql` to ensure that the correct version is applied in `init` mode.

Reply:
Oops, sorry. I always forget that. Thanks for pointing it out.

Reply:
Should be fixed now.